lib/streams/abstract-writable-stream.js

```diff
@@ -91,6 +108,14 @@ module.exports = class AbstractWritableStream extends stream.Writable {
                 return callback(err);
             }
 
+            if (this._filterFn && !this._filterFn(filename, lstats)) {
+                console.log(`Do not add file ${filename} because it does not pass a filter function`);
+                return callback();
+            }
+
+            console.log(`Add file ${filename} because it is time to do this`);
```

Comment:
> because it is time to do this

Чо?)

Выше мы пишем, что не прошёл функцию. А тут что-то про время, хотя мы в абстрактном классе. Думаю, часть «because it is time to do this» лишняя.